extensions/wikia/MercuryApi/models/MercuryApi.class.php

```diff
@@ -276,6 +274,24 @@ public function processCuratedContent( $data ) {
 		return $data;
 	}
 
+	public function processTrendingArticles( $data ) {
```

Comment:
We are creating here an abstraction here over fairly new API

I would like us to start to think if we can use API data directly, and process them on client side
and if really we believe it is necessary 
maybe we should just consider creating v2 of those resources?